itest/starboard/starboard_cli_test.go

```diff
@@ -303,10 +305,12 @@ var _ = Describe("Starboard CLI", func() {
 								kube.LabelResourceNamespace: Equal(podNamespace),
 							}),
 							"OwnerReferences": ConsistOf(metav1.OwnerReference{
-								APIVersion: "v1",
-								Kind:       "Pod",
-								Name:       podName,
-								UID:        pod.UID,
+								APIVersion:         "v1",
```

Comment:
I will create a custom Gomega matcher to assert valid VulnerabilityReports controlled by the given workload